src/main/java/utils/YamlConfigReader.java

```diff
@@ -0,0 +1,38 @@
+package utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+public class YamlConfigReader {
+    private static final Logger logger = LoggerFactory.getLogger(YamlConfigReader.class);
+
+    public static String loadStaticSourcePathFromYaml() {
+        try (InputStream input = new FileInputStream("config.yaml")) {
+            Yaml yaml = new Yaml();
+            Map<String, String> map = yaml.load(input);
+            return map.get("staticSourcePath");
+        } catch (IOException e) {
+            logger.error("Error loading YAML file", e);
+            return "./src/main/resources/static"; // default 리턴 값
+        }
+    }
+
+    public static int loadPortFromYaml() {
```

Comment:
```suggestion
    public static int getPort() {
```

길게 이름이 중복되는 건 좋지 않습니다. 이미 클래스 이름에 포함된 내용은 빼 주시고 public class를 사용하는 입장에서 필요한 이름으로 간결하게 줄여 주세요.